lib/eventboss/listener.rb

```diff
@@ -17,15 +17,15 @@ def postpone_by(time_in_secs)
     end
 
     module ClassMethods
-      def eventboss_options(opts)
-        source_app = opts[:source_app] ? "#{opts[:source_app]}-" : ""
-        event_name = opts[:event_name]
-        destination_app = opts[:destination_app]
+      attr_reader :options
 
-        ACTIVE_LISTENERS["#{source_app}#{event_name}"] = {
+      def eventboss_options(options)
+        @options = options.compact
```

Comment:
Shouldn't we somehow normalize it?

Reply:
IDK, do you think it would help with something??

Reply:
Let's leave it as it is.